### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated VP Finance & Operations across the handbook and standardized role name to VP Finance & Operations for consistency. 

Note: The content around strategy and MLT cadences is outdated but will be addressed in a separate PR.

### CODE REVIEW

File: operations/operations/company-agreements.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Company Agreements
 
-**Note: Pull Requests for updates to this page need to be signed off by both VP Finance & Ops and CEO in the PR process.**
+**Note: Pull Requests for updates to this page need to be signed off by both VP Finance and CEO in the PR process.**
```
Comment (justinegeffen) [inline operations/operations/company-agreements.md:3]:
```suggestion
**Note: Pull Requests for updates to this page need to be signed off by both VP Finance & Operations and CEO in the PR process.**
```

File: operations/operations/company-agreements.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # Company Agreements
 
-**Note: Pull Requests for updates to this page need to be signed off by both VP Finance & Ops and CEO in the PR process.**
+**Note: Pull Requests for updates to this page need to be signed off by both VP Finance & Operations and CEO in the PR process.**
 
 ## Who can sign on behalf of the company?
 
-Company agreements should only be signed by CEO, Ian Tien, or VP Finance & Operations, Aneal Vallurupalli with the following exceptions:
+Company agreements should only be signed by CEO, Ian Tien, or VP Finance, Tim Quock with the following exceptions:
```
Comment (justinegeffen) [inline operations/operations/company-agreements.md]:
```suggestion
Company agreements should only be signed by CEO, Ian Tien, or VP Finance & Operations, Tim Quock with the following exceptions:
```

File: operations/operations/company-agreements.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
 * If you are on-site and need to sign an NDA to enter a facility you may sign such an agreement.
 * The following agreements may also be signed by the Head of Legal, Jamie Hurewitz:
   * Non-Disclosure Agreements.
   * Data Privacy Agreements
-* The following agreements may also be signed by the Director of Accounting, Tim Quock.
+* The following agreements may also be signed by the VP Finance, Tim Quock.
```
Comment (justinegeffen) [inline operations/operations/company-agreements.md:13]:
```suggestion
* The following agreements may also be signed by the VP Finance & Operations, Tim Quock.
```

File: operations/operations/mlt-cadence/README.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ This section outlines:
 
 * Ian Tien: Co-Founder and CEO
 * Corey Hulen: Co-Founder and CTO
-* Aneal Vallurupalli: VP, Finance
+* Tim Quock: VP, Finance
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md:20]:
```suggestion
* Tim Quock: VP, Finance & Operations
```

**Review by jasonblais — CHANGES_REQUESTED** [review]
@justinegeffen Tim's title is in fact `VP, Finance`

File: company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract.md [hunk 5]
```diff
@@ -58,7 +58,7 @@ Once a PO is submitted, the request is routed to the [Procurement Team](mailto:p
 
 If you did not provide a contract in the PO submission and have received spend approval, ask your vendor contact to send you a **Word Document** of their contract so that Legal may review and if necessary, REDLINE the agreement. You may reference the legal review process outlined in the [Authorization Matrix](https://drive.google.com/a/mattermost.com/open?id=1fDIMiO0uydB_1zCUxZ4sGfSnBJ0P_49zbeQGgTqbYPI).
 
-All contracts are to address either Ian Tien \(CEO\) or Aneal Vallurupalli \(VP Finance\), and should include the following relevant information:
+All contracts are to address either Ian Tien \(CEO\) or Time Quock \(VP Finance & Operations\), and should include the following relevant information:
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract.md]:
```suggestion
All contracts are to address either Ian Tien \(CEO\) or Tim Quock \(VP Finance\), and should include the following relevant information:
```

Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract.md]:
```suggestion
All contracts are to address either Ian Tien \(CEO\) or Tim Quock \(VP Finance\), and should include the following relevant information:
```

File: company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract.md [hunk 6]
```diff
@@ -96,7 +96,7 @@ Once the contract is approved, the Procurement Team will prepare the [contract f
 
 1. Create a PDF of the fully approved contract.
 2. Director or above will initial in DocuSign.
-3. Contract sent to VP Finance or CEO for final signature in DocuSign.
+3. Contract sent to VP Finance & Operations or CEO for final signature in DocuSign.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract.md]:
```suggestion
3. Contract sent to VP Finance or CEO for final signature in DocuSign.
```

File: contributors/join-us/staff-recruiting/README.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ In order to continually increase what we offer to customers we need to continual
...
+Hiring plans are created twice a year, and reviewed every quarter. Hiring Managers and department leads can request additional headcount by completing a [New Requisition ID Request Form](https://forms.gle/ewYXCm1FZPAb6Pyr6). As part of this request form and only for non-backfill roles, Hiring Managers will be asked to submit an [Issue/Solution Proposal](https://handbook.mattermost.com/operations/operations/company-processes/issue-solution). Finance assigns all requisition IDs and the CEO (Ian Tien) and VP, Finance & Operations (Tim Quock) will need to approve ISP ahead of a new requisition ID assignment.
```
Comment (justinegeffen) [inline contributors/join-us/staff-recruiting/README.md]:
```suggestion
Hiring plans are created twice a year, and reviewed every quarter. Hiring Managers and department leads can request additional headcount by completing a [New Requisition ID Request Form](https://forms.gle/ewYXCm1FZPAb6Pyr6). As part of this request form and only for non-backfill roles, Hiring Managers will be asked to submit an [Issue/Solution Proposal](https://handbook.mattermost.com/operations/operations/company-processes/issue-solution). Finance assigns all requisition IDs and the CEO (Ian Tien) and VP Finance (Tim Quock) will need to approve ISP ahead of a new requisition ID assignment.
```

File: operations/finance/README.md [hunk 8]
```diff
@@ -8,8 +8,7 @@ Email: Spend@mattermost.com (Budget requests), Procurement@mattermost.com (Procu
 
 ## The team
 
-* Aneal Vallurupalli, VP of Finance and Operations
-* Tim Quock, Director of Finance
+* Tim Quock, VP of Finance and Operations
```
Comment (justinegeffen) [inline operations/finance/README.md]:
```suggestion
* Tim Quock, VP Finance
```

File: operations/operations/company-agreements.md [hunk 9]
```diff
@@ -1,16 +1,16 @@
 # Company Agreements
 
-**Note: Pull Requests for updates to this page need to be signed off by both VP Finance & Ops and CEO in the PR process.**
+**Note: Pull Requests for updates to this page need to be signed off by both VP Finance & Operations and CEO in the PR process.**
```
Comment (justinegeffen) [inline operations/operations/company-agreements.md]:
```suggestion
**Note: Pull Requests for updates to this page need to be signed off by both VP Finance and CEO in the PR process.**
```

File: operations/operations/company-agreements.md [hunk 10]
```diff
@@ -1,16 +1,16 @@
...
 
 ## Who can sign on behalf of the company?
 
-Company agreements should only be signed by CEO, Ian Tien, or VP Finance & Operations, Aneal Vallurupalli with the following exceptions:
+Company agreements should only be signed by CEO, Ian Tien, or VP Finance & Operations, Tim Quock with the following exceptions:
```
Comment (justinegeffen) [inline operations/operations/company-agreements.md]:
```suggestion
Company agreements should only be signed by CEO, Ian Tien, or VP Finance, Tim Quock, with the following exceptions:
```

File: operations/operations/company-agreements.md [hunk 11]
```diff
@@ -1,16 +1,16 @@
...
 
 * If you are on-site and need to sign an NDA to enter a facility you may sign such an agreement.
 * The following agreements may also be signed by the Head of Legal, Jamie Hurewitz:
   * Non-Disclosure Agreements.
   * Data Privacy Agreements
-* The following agreements may also be signed by the Director of Accounting, Tim Quock.
+* The following agreements may also be signed by the VP Finance & Operations, Tim Quock.
```
Comment (justinegeffen) [inline operations/operations/company-agreements.md]:
```suggestion
* The following agreements may also be signed by the VP Finance, Tim Quock.
```

File: operations/operations/mlt-cadence/README.md [hunk 12]
```diff
@@ -17,7 +17,7 @@ This section outlines:
 
 * Ian Tien: Co-Founder and CEO
 * Corey Hulen: Co-Founder and CTO
-* Aneal Vallurupalli: VP, Finance
+* Tim Quock: VP, Finance & Operations
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* Tim Quock: VP, Finance
```

File: operations/operations/company-policies/community-response-policy.md [hunk 13]
```diff
@@ -101,7 +101,7 @@ Below is the list of members mapped to each role mentioned as an approved respon
 * CEO: Ian Tien
 * CTO: Corey Hulen
 * VP Eng: Chris Overton
-* VP Finance: Aneal Vallurupalli
+* VP Finance & Operations: Tim Quock
```
Comment (justinegeffen) [inline operations/operations/company-policies/community-response-policy.md]:
```suggestion
* VP Finance: Tim Quock
```

**Review by jasonblais — DISMISSED** [review]

**Comment by justinegeffen** [comment]
@TQuock, @it33, following up on this for your approval as per the Handbook. Thank you! :)

**Review by jasonblais — APPROVED** [review]

**Comment by jasonblais** [comment]
@justinegeffen Approved on Tim and Ian's behalf

**Pull request merged**